rllib/utils/metrics/metrics_logger.py

```diff
@@ -131,21 +150,23 @@ def __contains__(self, key: Union[str, Tuple[str, ...]]) -> bool:
 
     def peek(
         self,
-        key: Union[str, Tuple[str, ...]],
-        *,
-        default: Optional[Any] = None,
+        key: Union[str, Tuple[str, ...], None] = None,
+        default=None,
+        compile: bool = True,
         throughput: bool = False,
     ) -> Any:
-        """Returns the (reduced) value(s) found under the given key or key sequence.
-
-        If `key` only reaches to a nested dict deeper in `self`, that
-        sub-dictionary's entire values are returned as a (nested) dict with its leafs
-        being the reduced peek values.
+        """Returns the reduced values found in this MetricsLogger.
```

Comment:
Could we describe here in all detail the difference between `peek()` and `peek(compile=False)`? So compile (single value) is the default behavior, just like in the existing code, correct? But if I set compile=False, I would get the actually underlying list of values? Let's add one example to the docstring to clarify this.

Reply:
Yes! Obviously, if there is no reduce method, we can't reduce to a single value and will return a list instead.

Reply:
Added!